src/components/nav-bar/nav-bar.less

```diff
@@ -0,0 +1,26 @@
+.page-header {
```

Comment:
We shouldn't be adding any less files at this point. This all needs to be done with styled-components. That should hopefully resolve the need to override any `ant` classes directly, we can't do that.

Reply:
Sorry so no less files at all?

Reply:
Correct, it should all be done with styled-components